app/src/main/java/ch/epfl/culturequest/ui/map/MapsFragment.java

```diff
@@ -121,6 +195,28 @@ public void onResume() {
         if (mMap != null) {
             getLocationPermission();
         }
+        getProfilePicture();
+    }
+
+    private void getProfilePicture(){
+        Picasso.get().load(Profile.getActiveProfile().getProfilePicture()).into(new Target() {
+            @Override
+            public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom from) {
+                Log.i("PICTURE", "PICTURE LOADED");
+                profilePicture = getCircularBitmap(bitmap);
+                drawPositionMarker(viewModel.getCurrentLocation().getValue());
+            }
+
+            @Override
+            public void onBitmapFailed(Exception e, Drawable errorDrawable) {
```

Comment:
What happens if it can't retrieve the profile picture, does it put a default one ?

Reply:
Actually for now if it cannot you'll just have a plain orange dot (Which is what is used for the picture contour actually)